lib/config.js

```diff
@@ -44,6 +39,14 @@ Config.FILENAME = 'firebase.json';
 Config.TARGETS = ['hosting', 'rules'];
 Config.LEGACY_HOSTING_KEYS = ['public', 'rewrites', 'redirects', 'headers', 'ignore'];
 
+Config.prototype.importLegacyKeys = function() {
+  Config.LEGACY_HOSTING_KEYS.forEach(function(key) {
+    if (this._src[key]) {
```

Comment:
I know this is just a copy / paste of the code, but will this have unintended effects if `this._src[key]` is falsy? Do we want to instead do an existence check here like `_.has(this._src, key)`?